hr_employee_age/models/hr_employee.py

```diff
@@ -33,9 +32,11 @@ class HrEmployee(models.Model):
     )
 
     @api.one
+    @api.depends('birthday')
     def _compute_age(self):
         if self.birthday:
-            dBday = datetime.strptime(self.birthday, OE_DFORMAT).date()
-            dToday = datetime.now().date()
-            self.age = dToday.year - dBday.year - ((
-                dToday.month, dToday.day) < (dBday.month, dBday.day))
+            dBday = fields.Date.from_string(self.birthday)
+            dToday = fields.Date.from_string(fields.Date.today())
+            self.age = relativedelta(dToday, dBday).years
```

Comment:
You can do directly the substraction of one date minus the other and you will get a relativedelta object.